From: Sam Macbeth <sam@cliqz.com>
Date: Fri, 18 Dec 2020 14:51:02 +0100
Subject: Default bookmarks

---
 browser/profile/bookmarks.inc | 18 +++++++++++++++++-
 1 file changed, 17 insertions(+), 1 deletion(-)

diff --git a/browser/profile/bookmarks.inc b/browser/profile/bookmarks.inc
index 1a1522b..24764f2 100755
--- a/browser/profile/bookmarks.inc
+++ b/browser/profile/bookmarks.inc
@@ -4,7 +4,7 @@
 #filter emptyLines
 
 # LOCALIZATION NOTE: The 'en-US' strings in the URLs will be replaced with
-# your locale code, and link to your translated pages as soon as they're 
+# your locale code, and link to your translated pages as soon as they're
 # live.
 
 #define bookmarks_title Marcadores
@@ -69,4 +69,20 @@
 # Nightly builds only, link title for https://planet.mozilla.org/
 #define planet Planet Mozilla
 
+# LOCALIZATION NOTE (ghostery_heading):
+# Ghostery links folder name
+#define ghostery_heading Ghostery
+
+# LOCALIZATION NOTE (ghostery_help):
+# link title for https://www.ghostery.com/support/
+#define ghostery_help Help
+
+# LOCALIZATION NOTE (ghostery_plans):
+# link title for https://www.ghostery.com/pricing/
+#define ghostery_plans Ghostery Plans
+
+# LOCALIZATION NOTE (ghostery_about_us):
+# link title for https://www.ghostery.com/about-ghostery/
+#define ghostery_about_us About us
+
 #unfilter emptyLines
-- 
2.29.2

