From: Sam Macbeth <sam@cliqz.com>
Date: Fri, 21 Aug 2020 13:35:37 +0200
Subject: Include addons in browser bundle

---
 browser/extensions/moz.build | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/browser/extensions/moz.build b/browser/extensions/moz.build
index 42961bd7d8..6294f9d0d5 100644
--- a/browser/extensions/moz.build
+++ b/browser/extensions/moz.build
@@ -13,6 +13,9 @@ DIRS += [
     "pictureinpicture",
     "proxy-failover",
     "search-detection",
+    "ghostery",
+    "ghostery-search",
+    "ghostery-newtab",
 ]
 
 if CONFIG["NIGHTLY_BUILD"]:
-- 
2.34.1

