From: Sam Macbeth <sam@cliqz.com>
Date: Fri, 21 Aug 2020 13:35:37 +0200
Subject: Include addons in browser bundle

---
 browser/extensions/moz.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/browser/extensions/moz.build b/browser/extensions/moz.build
index 0eb3c53e76..b37208ee92 100644
--- a/browser/extensions/moz.build
+++ b/browser/extensions/moz.build
@@ -4,4 +4,4 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-DIRS += ["doh-rollout", "formautofill", "screenshots", "webcompat", "report-site-issue"]
+DIRS += ["doh-rollout", "formautofill", "screenshots", "webcompat", "report-site-issue", "ghostery", "ghostery-search", "ghostery-newtab"]
-- 
2.25.1

