From: Sam Macbeth <sam@cliqz.com>
Date: Fri, 21 Aug 2020 13:35:37 +0200
Subject: Include ghostery extension in browser bundle

---
 browser/extensions/moz.build | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/browser/extensions/moz.build b/browser/extensions/moz.build
index f71ece63c4..974c48c612 100644
--- a/browser/extensions/moz.build
+++ b/browser/extensions/moz.build
@@ -9,5 +9,6 @@ DIRS += [
     'formautofill',
     'screenshots',
     'webcompat',
-    'report-site-issue'
+    'report-site-issue',
+    'ghostery'
 ]
-- 
2.28.0

