From: Sam Macbeth <sam@cliqz.com>
Date: Fri, 21 Aug 2020 13:35:37 +0200
Subject: Include ghostery extension in browser bundle

---
 browser/extensions/moz.build | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/browser/extensions/moz.build b/browser/extensions/moz.build
index 4c9fa789d1..ee2cfa1919 100644
--- a/browser/extensions/moz.build
+++ b/browser/extensions/moz.build
@@ -10,5 +10,6 @@ DIRS += [
     'pdfjs',
     'screenshots',
     'webcompat',
-    'report-site-issue'
+    'report-site-issue',
+    'ghostery'
 ]
-- 
2.28.0

