From: Sam Macbeth <sam@cliqz.com>
Date: Thu, 10 Sep 2020 17:52:19 +0200
Subject: Update text in about dialog

---
 browser/base/content/aboutDialog.xhtml | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/browser/base/content/aboutDialog.xhtml b/browser/base/content/aboutDialog.xhtml
index 69cbf846bb..e8d8bd6b5d 100644
--- a/browser/base/content/aboutDialog.xhtml
+++ b/browser/base/content/aboutDialog.xhtml
@@ -133,12 +133,12 @@
             </description>
           </vbox>
           <description class="text-blurb" id="communityDesc" data-l10n-id="community-2">
-            <label is="text-link" href="http://www.mozilla.org/" data-l10n-name="community-mozillaLink"></label>
+            <label is="text-link" href="https://www.ghostery.com/" data-l10n-name="community-mozillaLink"></label>
             <label is="text-link" useoriginprincipal="true" href="about:credits" data-l10n-name="community-creditsLink"></label>
           </description>
           <description class="text-blurb" id="contributeDesc" data-l10n-id="helpus">
             <label is="text-link" href="https://donate.mozilla.org/?utm_source=firefox&#38;utm_medium=referral&#38;utm_campaign=firefox_about&#38;utm_content=firefox_about" data-l10n-name="helpus-donateLink"></label>
-            <label is="text-link" href="http://www.mozilla.org/contribute/" data-l10n-name="helpus-getInvolvedLink"></label>
+            <label is="text-link" href="https://github.com/ghostery/user-agent-desktop" data-l10n-name="helpus-getInvolvedLink"></label>
           </description>
         </vbox>
       </vbox>
@@ -147,7 +147,7 @@
       <hbox pack="center">
         <label is="text-link" class="bottom-link" useoriginprincipal="true" href="about:license" data-l10n-id="bottomLinks-license"></label>
         <label is="text-link" class="bottom-link" useoriginprincipal="true" href="about:rights" data-l10n-id="bottomLinks-rights"></label>
-        <label is="text-link" class="bottom-link" href="https://www.mozilla.org/privacy/" data-l10n-id="bottomLinks-privacy"></label>
+        <label is="text-link" class="bottom-link" href="https://www.ghostery.com/about-ghostery/privacy-statements/" data-l10n-id="bottomLinks-privacy"></label>
       </hbox>
       <description id="trademark" data-l10n-id="trademarkInfo"></description>
     </vbox>
-- 
2.29.2

