From: Sam Macbeth <sam@cliqz.com>
Date: Thu, 10 Sep 2020 09:32:48 +0200
Subject: Remove bundled mozilla extensions

---
 browser/extensions/moz.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/browser/extensions/moz.build b/browser/extensions/moz.build
index b37208ee92..0702dd747f 100644
--- a/browser/extensions/moz.build
+++ b/browser/extensions/moz.build
@@ -4,4 +4,4 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-DIRS += ["doh-rollout", "formautofill", "screenshots", "webcompat", "report-site-issue", "ghostery", "ghostery-search", "ghostery-newtab"]
+DIRS += ["webcompat", "report-site-issue", "ghostery", "ghostery-search", "ghostery-newtab"]
-- 
2.25.1

