From: Sam Macbeth <sam@cliqz.com>
Date: Fri, 16 Oct 2020 15:47:47 +0200
Subject: Hide Protection Dashboard link from right burger menu

---
 browser/base/content/browser.xhtml | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/browser/base/content/browser.xhtml b/browser/base/content/browser.xhtml
index c2103dcc71..41c3d8abe5 100644
--- a/browser/base/content/browser.xhtml
+++ b/browser/base/content/browser.xhtml
@@ -758,8 +758,8 @@
                         closemenu="none"
                         oncommand="gSync.toggleAccountPanel(this, event)"/>
         </toolbaritem>
-        <toolbarseparator class="sync-ui-item"/>
         <!--
+        <toolbarseparator class="sync-ui-item"/>
         <toolbaritem>
           <toolbarbutton id="appMenu-protection-report-button"
                class="subviewbutton subviewbutton-iconic"
@@ -771,8 +771,8 @@
             </label>
           </toolbarbutton>
         </toolbaritem>
-        -->
         <toolbarseparator id="appMenu-tp-separator"/>
+        -->
         <toolbarbutton id="appMenu-new-window-button"
                        class="subviewbutton subviewbutton-iconic"
                        label="&newNavigatorCmd.label;"
@@ -1837,6 +1837,7 @@
                    class="chromeclass-toolbar-additional"/>
               <!-- Use onclick instead of normal popup= syntax since the popup
                    code fires onmousedown, and hence eats our favicon drag events. -->
+              <!--
               <box id="tracking-protection-icon-container" align="center"
                    role="button"
                    onclick="gProtectionsHandler.handleProtectionsButtonEvent(event);"
@@ -1851,6 +1852,7 @@
                   <description id="tracking-protection-icon-tooltip-label" class="tooltip-label"/>
                 </tooltip>
               </box>
+              -->
               <box id="identity-box"
                    pageproxystate="invalid"
                    ondragstart="gIdentityHandler.onDragStart(event);">
-- 
2.32.0

