From: Sam Macbeth <sam@cliqz.com>
Date: Fri, 16 Oct 2020 15:47:47 +0200
Subject: Hide Protection Dashboard link from right burger menu

---
 browser/base/content/browser.xhtml | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/browser/base/content/browser.xhtml b/browser/base/content/browser.xhtml
index ded22553bd..92ef5635f7 100644
--- a/browser/base/content/browser.xhtml
+++ b/browser/base/content/browser.xhtml
@@ -762,8 +762,8 @@
                         closemenu="none"
                         oncommand="gSync.toggleAccountPanel('PanelUI-fxa', this, event)"/>
         </toolbaritem>
-        <toolbarseparator class="sync-ui-item"/>
         <!--
+        <toolbarseparator class="sync-ui-item"/>
         <toolbaritem>
           <toolbarbutton id="appMenu-protection-report-button"
                class="subviewbutton subviewbutton-iconic"
@@ -775,8 +775,8 @@
             </label>
           </toolbarbutton>
         </toolbaritem>
-        -->
         <toolbarseparator id="appMenu-tp-separator"/>
+	-->
         <toolbarbutton id="appMenu-new-window-button"
                        class="subviewbutton subviewbutton-iconic"
                        label="&newNavigatorCmd.label;"
@@ -1827,6 +1827,7 @@
                    class="chromeclass-toolbar-additional"/>
               <!-- Use onclick instead of normal popup= syntax since the popup
                    code fires onmousedown, and hence eats our favicon drag events. -->
+              <!--
               <box id="tracking-protection-icon-container" align="center"
                    role="button"
                    onclick="gProtectionsHandler.handleProtectionsButtonEvent(event);"
@@ -1844,6 +1845,7 @@
                   <description id="tracking-protection-icon-tooltip-label" class="tooltip-label"/>
                 </tooltip>
               </box>
+	      -->
               <box id="identity-box"
                    pageproxystate="invalid"
                    ondragstart="gIdentityHandler.onDragStart(event);">
-- 
2.25.1

