From: Sam Macbeth <sam@cliqz.com>
Date: Wed, 23 Sep 2020 17:02:37 +0200
Subject: Allow setting service url to be overrided by prefs

---
 services/settings/Utils.sys.mjs | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/services/settings/Utils.sys.mjs b/services/settings/Utils.sys.mjs
index 0144fad99c..bd7609f6a1 100644
--- a/services/settings/Utils.sys.mjs
+++ b/services/settings/Utils.sys.mjs
@@ -95,9 +95,7 @@ function _isUndefined(value) {

 export var Utils = {
   get SERVER_URL() {
-    return lazy.allowServerURLOverride
-      ? lazy.gServerURL
-      : AppConstants.REMOTE_SETTINGS_SERVER_URL;
+    return lazy.gServerURL;
   },

   CHANGES_PATH: "/buckets/monitor/collections/changes/changeset",
diff --git a/toolkit/modules/AppConstants.sys.mjs b/toolkit/modules/AppConstants.sys.mjs
index b430666f68..97a7e7a27d 100644
--- a/toolkit/modules/AppConstants.sys.mjs
+++ b/toolkit/modules/AppConstants.sys.mjs
@@ -418,7 +418,7 @@ export var AppConstants = Object.freeze({
 #ifdef MOZ_THUNDERBIRD
     false,
 #else
-    true,
+    false,
 #endif

   REMOTE_SETTINGS_DEFAULT_BUCKET:
--
2.39.2 (Apple Git-143)

