From: Sam Macbeth <sam@cliqz.com>
Date: Wed, 23 Sep 2020 17:02:37 +0200
Subject: Allow setting service url to be overrided by prefs

---
 services/settings/Utils.jsm | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/services/settings/Utils.jsm b/services/settings/Utils.jsm
index f91e5dcb67..db0d7948be 100644
--- a/services/settings/Utils.jsm
+++ b/services/settings/Utils.jsm
@@ -88,9 +88,7 @@ function _isUndefined(value) {
 
 var Utils = {
   get SERVER_URL() {
-    return allowServerURLOverride
-      ? gServerURL
-      : "https://firefox.settings.services.mozilla.com/v1";
+    return gServerURL;
   },
 
   CHANGES_PATH: "/buckets/monitor/collections/changes/changeset",
-- 
2.35.1

