From: Sam Macbeth <sam@cliqz.com>
Date: Wed, 23 Sep 2020 17:02:37 +0200
Subject: Allow setting service url to be overrided by prefs

---
 services/settings/Utils.jsm | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/services/settings/Utils.jsm b/services/settings/Utils.jsm
index 549f8cc41d..8874fdfe94 100644
--- a/services/settings/Utils.jsm
+++ b/services/settings/Utils.jsm
@@ -60,12 +60,7 @@ var Utils = {
     );
     const isXpcshell = env.exists("XPCSHELL_TEST_PROFILE_DIR");
     const isNotThunderbird = AppConstants.MOZ_APP_NAME != "thunderbird";
-    return AppConstants.RELEASE_OR_BETA &&
-      !Cu.isInAutomation &&
-      !isXpcshell &&
-      isNotThunderbird
-      ? "https://firefox.settings.services.mozilla.com/v1"
-      : gServerURL;
+    return gServerURL;
   },
 
   CHANGES_PATH: "/buckets/monitor/collections/changes/changeset",
-- 
2.32.0

