From: Sam Macbeth <sam@cliqz.com>
Date: Fri, 23 Oct 2020 15:01:24 +0200
Subject: Allow update channel to be configured via prefs

---
 toolkit/modules/UpdateUtils.sys.mjs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/toolkit/modules/UpdateUtils.sys.mjs b/toolkit/modules/UpdateUtils.sys.mjs
index 5bd258b520..ad825c0b9f 100644
--- a/toolkit/modules/UpdateUtils.sys.mjs
+++ b/toolkit/modules/UpdateUtils.sys.mjs
@@ -38,7 +38,7 @@ export var UpdateUtils = {
    *        Whether or not to include the partner bits. Default: true.
    */
   getUpdateChannel(aIncludePartners = true) {
-    let defaults = Services.prefs.getDefaultBranch(null);
+    let defaults = Services.prefs;
     let channel = defaults.getCharPref(
       "app.update.channel",
       AppConstants.MOZ_UPDATE_CHANNEL
diff --git a/browser/components/enterprisepolicies/Policies.sys.mjs b/browser/components/enterprisepolicies/Policies.sys.mjs
index ed183f7def..69284bf149 100644
--- a/browser/components/enterprisepolicies/Policies.sys.mjs
+++ b/browser/components/enterprisepolicies/Policies.sys.mjs
@@ -1903,7 +1903,7 @@ export var Policies = {
         "security.warn_submit_secure_to_insecure",
       ];
       const blockedPrefs = [
-        "app.update.channel",
+        // "app.update.channel",
         "app.update.lastUpdateTime",
         "app.update.migrated",
         "browser.vpn_promo.disallowed_regions",
--
2.39.2 (Apple Git-143)

