From: Krzysztof Jan Modras <chrmod@chrmod.net>
Date: Wed, 18 Nov 2020 13:52:24 +0100
Subject: Disable ion / pioneer studies

fix #358
---
 browser/components/BrowserGlue.jsm | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/browser/components/BrowserGlue.jsm b/browser/components/BrowserGlue.jsm
index 7f69db89e8..a46fa8605b 100644
--- a/browser/components/BrowserGlue.jsm
+++ b/browser/components/BrowserGlue.jsm
@@ -2313,8 +2313,10 @@ BrowserGlue.prototype = {
     this._monitorScreenshotsPref();
     this._monitorWebcompatReporterPref();
     this._monitorHTTPSOnlyPref();
+    /*
     this._monitorIonPref();
     this._monitorIonStudies();
+    */
     this._setupSearchDetection();
 
     if (AppConstants.NIGHTLY_BUILD) {
-- 
2.34.1

