From: Krzysztof Jan Modras <chrmod@chrmod.net>
Date: Wed, 30 Sep 2020 12:22:19 +0200
Subject: Move Ghostery icon to UrlBar

---
 browser/components/customizableui/CustomizableUI.jsm | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/browser/components/customizableui/CustomizableUI.jsm b/browser/components/customizableui/CustomizableUI.jsm
index 517273b89c..8ee7598c52 100644
--- a/browser/components/customizableui/CustomizableUI.jsm
+++ b/browser/components/customizableui/CustomizableUI.jsm
@@ -1440,6 +1440,12 @@ var CustomizableUIInternal = {
       this.ensureButtonContextMenu(widgetNode, aAreaNode);
     }
 
+    if (widgetNode.getAttribute("data-extensionid") === "firefox@ghostery.com") {
+      const container = window.document.getElementById("urlbar-input-container");
+      container.insertAdjacentElement('afterbegin', widgetNode);
+      return;
+    }
+
     let [insertionContainer, nextNode] = this.findInsertionPoints(
       widgetNode,
       aAreaNode
-- 
2.28.0

