From: Sam Macbeth <sam@cliqz.com>
Date: Thu, 3 Dec 2020 13:55:50 +0100
Subject: OpenURL on buildID changes

---
 browser/components/BrowserContentHandler.jsm | 10 ++--------
 1 file changed, 2 insertions(+), 8 deletions(-)

diff --git a/browser/components/BrowserContentHandler.jsm b/browser/components/BrowserContentHandler.jsm
index 47647e8e91..6ccd150c02 100644
--- a/browser/components/BrowserContentHandler.jsm
+++ b/browser/components/BrowserContentHandler.jsm
@@ -688,13 +688,13 @@ nsBrowserContentHandler.prototype = {
             // we may open the startPage in addition to restoring the session.
             willRestoreSession = lazy.SessionStartup.isAutomaticRestoreEnabled();
 
+          case OVERRIDE_NEW_BUILD_ID:
             overridePage = Services.urlFormatter.formatURLPref(
               "startup.homepage_override_url"
             );
             let update = lazy.UpdateManager.readyUpdate;
             if (
-              update &&
-              Services.vc.compare(update.appVersion, old_mstone) > 0
+              update
             ) {
               overridePage = getPostUpdateOverridePage(update, overridePage);
               // Send the update ping to signal that the update was successful.
@@ -703,12 +703,6 @@ nsBrowserContentHandler.prototype = {
 
             overridePage = overridePage.replace("%OLD_VERSION%", old_mstone);
             break;
-          case OVERRIDE_NEW_BUILD_ID:
-            if (lazy.UpdateManager.readyUpdate) {
-              // Send the update ping to signal that the update was successful.
-              lazy.UpdatePing.handleUpdateSuccess(old_mstone, old_buildId);
-            }
-            break;
         }
       }
     } catch (ex) {}
-- 
2.36.0

