From: Sam Macbeth <sam@cliqz.com>
Date: Thu, 3 Dec 2020 13:55:50 +0100
Subject: OpenURL on buildID changes

---
 browser/components/BrowserContentHandler.sys.mjs | 10 ++--------
 1 file changed, 2 insertions(+), 8 deletions(-)

diff --git a/browser/components/BrowserContentHandler.sys.mjs b/browser/components/BrowserContentHandler.sys.mjs
index 18f3dd6ec9..5c7def8b6b 100644
--- a/browser/components/BrowserContentHandler.sys.mjs
+++ b/browser/components/BrowserContentHandler.sys.mjs
@@ -737,14 +737,13 @@ nsBrowserContentHandler.prototype = {
             // we may open the startPage in addition to restoring the session.
             willRestoreSession =
               lazy.SessionStartup.isAutomaticRestoreEnabled();
-
+          case OVERRIDE_NEW_BUILD_ID:
             overridePage = Services.urlFormatter.formatURLPref(
               "startup.homepage_override_url"
             );
             let update = lazy.UpdateManager.readyUpdate;
             if (
-              update &&
-              Services.vc.compare(update.appVersion, old_mstone) > 0
+              update
             ) {
               overridePage = getPostUpdateOverridePage(update, overridePage);
               // Send the update ping to signal that the update was successful.
@@ -754,13 +753,6 @@ nsBrowserContentHandler.prototype = {

             overridePage = overridePage.replace("%OLD_VERSION%", old_mstone);
             break;
-          case OVERRIDE_NEW_BUILD_ID:
-            if (lazy.UpdateManager.readyUpdate) {
-              // Send the update ping to signal that the update was successful.
-              lazy.UpdatePing.handleUpdateSuccess(old_mstone, old_buildId);
-              lazy.LaterRun.enable(lazy.LaterRun.ENABLE_REASON_UPDATE_APPLIED);
-            }
-            break;
         }
       }
     } catch (ex) {}
--
2.39.2 (Apple Git-143)

