From: Sam Macbeth <sam@cliqz.com>
Date: Thu, 3 Dec 2020 13:55:50 +0100
Subject: OpenURL on buildID changes

---
 browser/components/BrowserContentHandler.sys.mjs | 10 ++--------
 1 file changed, 2 insertions(+), 8 deletions(-)

diff --git a/browser/components/BrowserContentHandler.sys.mjs b/browser/components/BrowserContentHandler.sys.mjs
index ca7cf4d2c4..f3539822ad 100644
--- a/browser/components/BrowserContentHandler.sys.mjs
+++ b/browser/components/BrowserContentHandler.sys.mjs
@@ -807,6 +807,7 @@ nsBrowserContentHandler.prototype = {
             // Turn on 'later run' pages for new profiles.
             lazy.LaterRun.enable(lazy.LaterRun.ENABLE_REASON_NEW_PROFILE);
             break;
+          case OVERRIDE_NEW_BUILD_ID:
           case OVERRIDE_NEW_MSTONE: {
             // Check whether we will restore a session. If we will, we assume
             // that this is an "update" session. This does not take crashes
@@ -859,8 +860,7 @@ nsBrowserContentHandler.prototype = {
             }

             if (
-              update &&
-              Services.vc.compare(update.appVersion, old_mstone) > 0
+              update
             ) {
               overridePage = getPostUpdateOverridePage(
                 update,
@@ -892,13 +892,6 @@ nsBrowserContentHandler.prototype = {
             overridePage = overridePage.replace("%OLD_VERSION%", old_mstone);
             break;
           }
-          case OVERRIDE_NEW_BUILD_ID:
-            if (lazy.UpdateManager.readyUpdate) {
-              // Send the update ping to signal that the update was successful.
-              lazy.UpdatePing.handleUpdateSuccess(old_mstone, old_buildId);
-              lazy.LaterRun.enable(lazy.LaterRun.ENABLE_REASON_UPDATE_APPLIED);
-            }
-            break;
         }
       }
     } catch (ex) {}

--
2.39.2 (Apple Git-143)

