From: Krzysztof Jan Modras <chrmod@chrmod.net>
Date: Wed, 18 Nov 2020 13:52:24 +0100
Subject: Disable ion / pioneer studies

fix #358
---
 browser/components/BrowserGlue.jsm | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/browser/components/BrowserGlue.jsm b/browser/components/BrowserGlue.jsm
index 5324eecaf6..07cd25d66b 100644
--- a/browser/components/BrowserGlue.jsm
+++ b/browser/components/BrowserGlue.jsm
@@ -2212,8 +2212,8 @@ BrowserGlue.prototype = {
     this._monitorScreenshotsPref();
     this._monitorWebcompatReporterPref();
     this._monitorHTTPSOnlyPref();
-    this._monitorIonPref();
-    this._monitorIonStudies();
+    // this._monitorIonPref();
+    // this._monitorIonStudies();
     if (AppConstants.NIGHTLY_BUILD) {
       this._monitorTranslationsPref();
     }
-- 
2.32.0

