From: Krzysztof Jan Modras <chrmod@chrmod.net>
Date: Wed, 18 Nov 2020 13:52:24 +0100
Subject: Disable ion / pioneer studies

fix #358
---
 browser/components/BrowserGlue.jsm | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/browser/components/BrowserGlue.jsm b/browser/components/BrowserGlue.jsm
index 869a90ef40..c3af1a2d76 100644
--- a/browser/components/BrowserGlue.jsm
+++ b/browser/components/BrowserGlue.jsm
@@ -2426,8 +2426,8 @@ BrowserGlue.prototype = {
     this._monitorScreenshotsPref();
     this._monitorWebcompatReporterPref();
     this._monitorHTTPSOnlyPref();
-    this._monitorIonPref();
-    this._monitorIonStudies();
+    // this._monitorIonPref();
+    // this._monitorIonStudies();
 
     let pService = Cc["@mozilla.org/toolkit/profile-service;1"].getService(
       Ci.nsIToolkitProfileService
-- 
2.29.2

