From: Jenkins <jenkins@magrathea>
Date: Tue, 14 Feb 2023 20:32:16 +0100
Subject: Build: mach package use MOZ_FETCHES in automation

---
 build/moz.configure/bootstrap.configure | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/build/moz.configure/bootstrap.configure b/build/moz.configure/bootstrap.configure
index 5abb27f469..e34e9472cf 100644
--- a/build/moz.configure/bootstrap.configure
+++ b/build/moz.configure/bootstrap.configure
@@ -146,6 +146,7 @@ def bootstrap_path(path, **kwargs):

     @depends(
         enable_bootstrap,
+        "MOZ_AUTOMATION",
         toolchains_base_dir,
         moz_fetches_dir,
         bootstrap_toolchain_tasks,
@@ -164,6 +165,7 @@ def bootstrap_path(path, **kwargs):
     @imports(_from="__builtin__", _import="Exception")
     def bootstrap_path(
         bootstrap,
+        automation,
         toolchains_base_dir,
         moz_fetches_dir,
         tasks,
@@ -294,7 +296,7 @@ def bootstrap_path(path, **kwargs):
             return True

         path = os.path.join(toolchains_base_dir, path_prefix, *path_parts)
-        if bootstrap and bootstrap(path_parts[0]):
+        if bootstrap and bootstrap(path_parts[0]) and not automation:
             try:
                 if not try_bootstrap(os.path.exists(path)):
                     # If there aren't toolchain artifacts to use for this build,
--
2.39.2 (Apple Git-143)

