From: Jenkins <jenkins@magrathea>
Date: Tue, 14 Feb 2023 20:32:16 +0100
Subject: Build: mach package use MOZ_FETCHES in automation

---
 build/moz.configure/bootstrap.configure | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/build/moz.configure/bootstrap.configure b/build/moz.configure/bootstrap.configure
index d8deddbb9e..a556c7f2a2 100644
--- a/build/moz.configure/bootstrap.configure
+++ b/build/moz.configure/bootstrap.configure
@@ -120,6 +120,7 @@ def bootstrap_path(path, **kwargs):

     @depends(
         "--enable-bootstrap",
+        "MOZ_AUTOMATION",
         want_bootstrap,
         toolchains_base_dir,
         moz_fetches_dir,
@@ -139,6 +140,7 @@ def bootstrap_path(path, **kwargs):
     @imports(_from="__builtin__", _import="Exception")
     def bootstrap_path(
         enable_bootstrap,
+        automation,
         want_bootstrap,
         toolchains_base_dir,
         moz_fetches_dir,
@@ -275,7 +277,7 @@ def bootstrap_path(path, **kwargs):
             try:
                 # With --enable-bootstrap=no-update, we don't `try_bootstrap`, except
                 # when the toolchain can't be found.
-                if (
+                if not automation and (
                     "no-update" not in enable_bootstrap or not exists
                 ) and not try_bootstrap(exists):
                     # If there aren't toolchain artifacts to use for this build,

--
2.39.2 (Apple Git-143)

