From: Sam Macbeth <sam@cliqz.com>
Date: Thu, 3 Dec 2020 13:55:50 +0100
Subject: Force openURL on all updates

---
 browser/components/BrowserContentHandler.jsm | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/browser/components/BrowserContentHandler.jsm b/browser/components/BrowserContentHandler.jsm
index 706962c9f6..9c69e0488e 100644
--- a/browser/components/BrowserContentHandler.jsm
+++ b/browser/components/BrowserContentHandler.jsm
@@ -678,8 +678,7 @@ nsBrowserContentHandler.prototype = {
             );
             let update = UpdateManager.readyUpdate;
             if (
-              update &&
-              Services.vc.compare(update.appVersion, old_mstone) > 0
+              update
             ) {
               overridePage = getPostUpdateOverridePage(update, overridePage);
               // Send the update ping to signal that the update was successful.
-- 
2.29.2

